### PR TITLE
Adds warning if you're on staging.ninja

### LINKED ISSUE #1078: Warning Banner for Staging Environment
At the moment we have a public staging environment. I have no issue with this in principle as we don't test much security-conscious code, but we should warn people who are on the staging domain that uptime is not guaranteed, we do not support it for general use, it is likely bleeding edge code and it may be taken down / brought up at no notice.

The banner should link them to uclapi.com. We could also have a watermark or such on the wallpaper that says "Staging" or similar.

Open to design tips on this! ✨

### PR DESCRIPTION
**Pull request by zipy124**
## What does this PR do?
Adds a banner at the top to warn you that you are on our staging unstable branch.

## ✅ Pull Request checklist

- [ ] Is this code complete?
- [ ] Are tests done/modified?
- [ ] Are docs written for this PR? (if applicable)

## 🚨 Is this a breaking change for API clients?
no

## :squirrel: Deploy notes
re-build front-end provided this isn't done automatically?

## Anything else
closes #1078

### CODE REVIEW

**Comment by zipy124** [comment]
Example of how it looks: 
screenshot from 2019-02-26 22-03-17
Please give suggestions on how we want it to look?

File: frontend/src/components/getStarted/StagingBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+
+export default class StagingBanner extends React.Component {
+
+  render() {
+    return (
+      <div className="stagingBanner">
+        <div className="container">
+          <h1>Warning! This is our testing domain, for our stable API please go to</h1>
```
Comment (ChristopherHammond13) [inline frontend/src/components/getStarted/StagingBanner.jsx]:
```suggestion
          <h1>Warning! This is our bleeding-edge staging environment, and therefore performance, accuracy and reliability of the API cannot be guaranteed. For our stable, supported API please go to</h1>
```

File: frontend/src/components/getStarted/getStarted.jsx [hunk 2]
```diff
@@ -7,13 +7,17 @@ import Demo from './Demo.jsx';
...
 export default class GetStartedComponent extends React.Component {
-
+    constructor(props){
+      super(props);
+      this.host = window.location.hostname;
+    }
     render () {
       return (
         <div>
+          {this.host == "staging.ninja" && <StagingBanner />}
```
Comment (ChristopherHammond13) [inline frontend/src/components/getStarted/getStarted.jsx]: This is a sexy as hell one-liner

Comment (zipy124) [reply frontend/src/components/getStarted/getStarted.jsx]: A compliment on my front end code? *swoons*

Comment (ChristopherHammond13) [reply frontend/src/components/getStarted/getStarted.jsx]: hell to da yeh!

**Pull request merged**